apps/api/.env.example

```diff
@@ -1,27 +1,29 @@
 # API configuration
-HOST=
-PORT=
-API_KEY=
+HOST= ;e.g. localhost
+PORT= ;e.g. 3001
+API_KEY= ;any string, e.g. some_api_key
 
 # Postgres configuration
-DATABASE_URL=
+DATABASE_URL= ;match the DB you’ve created, e.g. postgres://postgres:postgres@localhost:5432/algomart
 DATABASE_SCHEMA=public
 
 # Algod configuration
-ALGOD_SERVER=
-ALGOD_PORT=
-ALGOD_TOKEN=
-ALGOD_ENV=testnet
+ALGOD_SERVER= ;e.g. http://localhost
+ALGOD_PORT= ;e.g. 4001
+ALGOD_TOKEN= ;e.g. aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+ALGOD_ENV= ;e.g. testnet
```

Comment:
Someone new to this might not realize that something like that ALGOD_TOKEN is actually valid (for dev). Is there a downside to just defining the dev defaults as opposed to showing the `;e.g.`?

Reply:
Not that I'm aware and I'm happy to update if there are no objections